stripe/src/main/java/com/stripe/android/model/ConfirmSetupIntentParams.kt

```diff
@@ -112,17 +113,41 @@ data class ConfirmSetupIntentParams internal constructor(
          * to the SetupIntent.
          *
          * @param clientSecret The client secret of this SetupIntent. Used for client-side retrieval using a publishable key.
+         *
+         * @return params that can be use to confirm a SetupIntent
+         */
+        @JvmStatic
+        fun createWithoutPaymentMethod(
+            clientSecret: String
+        ): ConfirmSetupIntentParams {
+            return ConfirmSetupIntentParams(
+                clientSecret = clientSecret
+            )
+        }
+
+        /**
+         * Create the parameters necessary for confirming a SetupIntent, without specifying a payment method
+         * to attach to the SetupIntent. Only use this if a payment method has already been attached
+         * to the SetupIntent.
+         *
+         * @deprecated See [Stripe.confirmSetupIntent] for more details on `return_url`.
```

Comment:
seems `Stripe.confirmSetupIntent` isn't mentioning anything about it?

Reply:
See #3655